app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
@Jiri-Kremser yes, the Alerts client expects the tenant from the options, the default 'hawkular' tenant used by Middleware is set in the upper raw_client as far as I remember. Inside the Alerts rb there is not tenant manipulation.